static/stylesheets/application.css

```diff
@@ -936,7 +936,7 @@
 }
 
 .govuk-width-container {
-  max-width: 960px;
```

Comment:
don't think we should be including raw css file, should be using scss instead

Reply:
There's a .gitignore in place for this.  It squeezed through earlier but looks ok now removed.